.gitignore

```diff
@@ -4,4 +4,7 @@ package-lock.json
 junit-reports
 
 build/
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+test-e2e/output
+test-e2e/staging
```

Comment:
```suggestion
test-e2e/staging

```

Reply:
Both directories should be ignored: codeceptjs dumps errors and captured traffic in `output`, but it has a random suffix, so we move the captured requests into `staging` and renamed them to a canonical name.

Reply:
oh no, I got that
this suggestion is just an empty line at the end of the file